redundancygroup.spec

```diff
@@ -171,6 +171,33 @@
             "default_order": false,
             "default_value": null,
             "deprecated": false,
+            "description": "The gateway configuration owner in this Redundant Group. when Redundant Group is deleted this gateway will recieve vport associations ",
```

Comment:
This description is wrong compared to the one in the API docs (it talks about peers). (i opened PR #5 today to fix this particular missing attribute ;))